server/src/main/kotlin/suwayomi/tachidesk/manga/model/dataclass/MangaDataClass.kt

```diff
@@ -33,11 +34,16 @@ data class MangaDataClass(
     val meta: Map<String, String> = emptyMap(),
 
     val realUrl: String? = null,
+    var lastFetchedAt: Long? = 0,
+    var chaptersLastFetchedAt: Long? = 0,
 
     val freshData: Boolean = false,
     var unreadCount: Int? = null,
     var downloadCount: Int? = null,
-    var chapterCount: Int? = null
+    var chapterCount: Int? = null,
+
+    val age: Long? = if (lastFetchedAt == null) 0 else Instant.now().epochSecond.minus(lastFetchedAt),
+    val chaptersAge: Long? = if (chaptersLastFetchedAt == null) null else Instant.now().epochSecond.minus(chaptersLastFetchedAt)
```

Comment:
why should the server supply these values?

Reply:
Theoretically, there could be issue if server and client have different time. These values would make it so that client does not have to calculate the value itself. Realistically, it probably won't ever be an issue.

I thought there would be a better way to add these values to API, but I could not find it. I would prefer if the values were calculated later and not stored on MangaDataClass. Maybe as a getter or something. But I don't know kotlin enough and it would not allow me to add them as getters.